src/test/java/core/basesyntax/RegistrationValidationTests.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+
+public class RegistrationValidationTests {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Figure out which way is better to initialize your RegistrationServiceImpl class instance: @BeforeEach or @BeforeAll, what’s the difference?

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/junit/registration-validation

Reply:
Its implemented before all as single instance. IntelliJ does not allow to make annotation because it is field. Would there be a particular reason announce this object each time?